lib/pages/editor-page.js

```diff
@@ -140,4 +140,16 @@ export default class EditorPage extends BaseContainer {
 	titleShown() {
 		return this.driver.findElement( by.css( '.editor-title__input' ) ).getAttribute( 'value' );
 	}
+
+	publishEnabled() {
+		const publishSelector = by.css( '.editor-publish-button' );
+		return this.driver.findElement( publishSelector ).getAttribute( 'disabled' ).then( ( d ) => {
+			return d !== 'true';
+		} );
+	}
+
+	emailVerificationNotice() {
```

Comment:
This should have a more meaningful name like `emailVerificationNoticeShown()` or `emailVerificationNoticeDisplayed()`